tck/src/main/java/ee/jakarta/tck/concurrent/spec/ContextService/contextPropagate/ContextPropagationTests.java

```diff
@@ -259,6 +259,10 @@ public void testContextualConsumer() throws Throwable {
      * A ContextService contextualizes a Function, which can be supplied as a dependent stage action
      * to an unmanaged CompletableFuture. The dependent stage action runs with the thread context of
      * the thread that contextualizes the Function, per the configuration of the ContextServiceDefinition.
+     *
+     * Assertions on results[0] and results[1] are both invalid because treating those two UNCHANGED context types as
+     * though they were CLEARED.
```

Comment:
This comment seems out of place given that this pull is attempting to correct the assertion on results[0] to make it valid.